jekyll/_components/tables.md

```diff
@@ -2,17 +2,142 @@
+
+Filter controls are provided in the form of buttons for specific resource types and a filter input field for filtering URLs.
+</div>
+</div>
+
 ## Styles
 
+Table rows alternate their background color, the subtle "zebra" effect assists in legibility:
+
+### Dark mode
+<div class="grid-2" markdown="1">
+![Usage Example of a Checkbox as used in Firefox Options](../images/components/tables/table-grid-dark.png)
+
+<div markdown="1">
+Odd rows: Grey 80 `#2a2a2e`
+
+Even rows: `rgba(255,255,255,0.05)`
+
+Column header: Grey 85 `#1b1b1d`
+
+Borders/splitters: `#3c3c3d`
+
+Text: Grey 40 `#b1b1b3`
+
+Selected row: `#204E8A`
+</div>
+</div>
+
+### Light mode
+<div class="grid-2" markdown="1">
+![Usage Example of a Checkbox as used in Firefox Options](../images/components/tables/table-grid-light.png)
+
+<div markdown="1">
+Odd rows: white `#ffffff`
+
+Even rows: `rgba(0,0,0,0.05)`
+
+Column header: Grey 10 `#f9f9fa`
+
+Borders/splitters: Grey 25 `#e0e0e2`
+
+Text: Grey 70 `#38383d`
+
+Selected row: Blue 55 `#0074e8`
+</div>
+</div>
+
+
+<div class="grid-2" markdown="1">
+![Table size and font properties](../images/components/tables/table-size-properties.svg)
+<div markdown="1">
+Font size: `11px`
+
+Row height: `24px`
+
+Line height: `24px`
+</div>
+</div>
+
 ## Behaviours
+
+### Sorting
+<div class="grid-2" markdown="1">
+![Table Column headers](../images/components/tables/table-header-button.png)
+
+<div markdown="1">
+Tables can posses sortable column headers, clicking these buttons will toggle the sorting of the column they relate to. In the image, clicking the "Size" button header toggles the Size column from largest to smallest and from smallest to largest.
+
+Selected column header (Light mode): Blue 55 `#0074e8`
+
+Selected column header (Dark mode): `#204E8A`
+</div>
+</div>
+
+<div class="grid-2" markdown="1">
+![Table Column headers](../images/components/tables/table-header-tooltip.png)
+
+<div markdown="1">
+If the table header is a clickable button to provide sorting controls, the tooltip should illustrate what the current sorting pattern is e.g. Sorted ascending, Sorted descending. 
+</div>
+</div>
+
+### Rich tooltips and previews
+
+<div class="grid-2" markdown="1">
+![Example of an image preview as tooltip](../images/components/tables/table-rich-tooltips.png)
+
+<div markdown="1">
+Table cells tooltips have the opportunity to provide extra information/context to the data being hovered over. For example, a domain name when hovered can additionally provide the IP address of the domain.
+
+When an image name is hovered over a [doorhanger](./doorhangers) should present a preview of the image with a caption containing the real pixel dimensions.
+
+</div>
+</div>
+
+## Future Improvements
+
+### Column resizing
+The Network table in particular requires individual column resizing capabilities so that lengthy strings like filenames and domains can be read easily without the need to alter the devtools or browser window width. 
```

Comment:
Can we say that all tables _should_ provide resizing and sorting, but some currently don't?